apps/jetpack-cloud/src/index.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * Internal dependencies
+ */
+import 'landing/jetpack-cloud';
```

Comment:
This currently loads and builds the entire entry point, but I'm not sure if we want to do that. Should we narrow it down to a particular location in the entry point?